modules/misc/news.nix

```diff
@@ -192,6 +192,19 @@ in
           A new service is available: 'services.screen-locker'.
         '';
       }
+
+      {
+        time = "2017-09-22T12:09:01+00:00";
+        condition = isString config.programs.git.extraConfig;
+        message = ''
+          The 'config.programs.git.extraConfig' parameter is now
```

Comment:
I think this should be just "The 'programs.git.extraConfig' parameter is now …". I.e., without the "config." part.